src/layout/HeroHeaderLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex } from '@chakra-ui/react'
+import { PropsWithChildren, ReactNode } from 'react'
+import { FallbackAccountImg } from '~constants'
+import { OrganizationImage } from '@vocdoni/chakra-components'
+
+/**
+ * Layout used to create a Hero banner page with the logo of the organization
+ * @param header - The header component, usually de ElectionHeader or the OrganizationHeader
+ * @param children - The content of the page under the lofo of the organization
+ */
+export const HeroHeaderLayout = ({ header, children }: { header: ReactNode } & PropsWithChildren) => {
```

Comment:
PropsWithChildren accepts arguments 😃 
```suggestion
export const HeroHeaderLayout = ({ header, children }: PropsWithChildren<{ header: ReactNode }>) => {
```